rascal-vscode-extension/src/test/vscode-suite/dsl.test.ts

```diff
@@ -47,13 +47,14 @@ describe('DSL', function () {
         const repl = new RascalREPL(bench, driver);
         await repl.start();
         await repl.execute("import demo::lang::pico::LanguageServer;");
-        repl.execute("main();"); // we don't wait, be cause we might miss pico loading window
+        const replExecuteMain = repl.execute("main();"); // we don't wait yet, because we might miss pico loading window
         const ide = new IDEOperations(browser);
         const isPicoLoading = ide.statusContains("Pico");
         await driver.wait(isPicoLoading, Delays.slow, "Pico DSL should start loading");
-        await repl.terminate();
-        // now wait for the Pico loader to dissapear
+        // now wait for the Pico loader to disappear
         await driver.wait(async () => !(await isPicoLoading()), Delays.extremelySlow, "Pico DSL should be finished starting", 100);
+        await replExecuteMain;
```

Comment:
a bit strange, as line 55 can only be true after the `main()` function has executed. are we sure this is needed?